src/lib/browse/LocalPlanningAuthoritiesLayerControl.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import {
+    hoveredToggle,
+    overwriteLineLayer,
+    overwritePmtilesSource,
+    overwritePolygonLayer,
+  } from "../../maplibre_helpers";
+  import { map } from "../../stores";
+  import {
+    ColorLegend,
+    ExternalLink,
+    HelpButton,
+    InteractiveLayer,
+  } from "../common";
+  import { Checkbox } from "../govuk";
+  import { colors } from "./colors";
+
+  let name = "local_planning_authorities";
+  let color = colors.local_planning_authorities;
+  let outlineLayer = `${name}-outline`;
+
+  overwritePmtilesSource(
+    $map,
+    name,
+    `https://atip.uk/layers/v1/${name}.pmtiles`
+  );
+
+  overwritePolygonLayer($map, {
+    id: name,
+    source: name,
+    sourceLayer: name,
+    color,
+    opacity: hoveredToggle(0.5, 0.1),
+  });
+  overwriteLineLayer($map, {
+    id: outlineLayer,
+    source: name,
+    sourceLayer: name,
+    color,
+    width: 2.5,
+  });
+
+  let show = false;
+  // InteractiveLayer manages the polygon layer, but we also need to control the outline
+  $: {
+    if ($map.getLayer(outlineLayer)) {
+      $map.setLayoutProperty(
+        outlineLayer,
+        "visibility",
+        show ? "visible" : "none"
+      );
+    }
+  }
+
+  // TODO Note there are overlapping features. InteractiveLayer only plumbs
```

Comment:
We could improve this, at the moderate cost of making all the other users of this layer more complex. I'd almost want to use a distinct component with the more complicated API. For now, we don't even know if this is the correct layer to show, or what we want to do with it, so let's not pay that cost just yet